extensions/amp-iframe/0.1/amp-iframe.js

```diff
@@ -237,6 +238,18 @@ export class AmpIframe extends AMP.BaseElement {
       }
     }
 
+
+
+    const isFixedAd = isPositionFixed(this.element, this.win)
```

Comment:
`isAdLike()` looks less expensive than `isPositionFixed()`, let's swap the order.